src/components/Favicon/Auto/hooks/useFaviconGrabbers.js

```diff
@@ -0,0 +1,55 @@
+import UrlService from '../../../../services/UrlService';
+import useCorsProxy from './useCorsProxy';
```

Comment:
`useCorsProxy` misleading naming. It follows the hook pattern, but it is not a React Hook.

Reply:
Ok, made it as a private static method of the newly created `FaviconGrabberService`